[2025-11-01T14:34:38.381+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: silver_transformation_pipeline.dbt_run_merchants manual__2025-11-01T14:34:36.691113+00:00 [queued]>
[2025-11-01T14:34:38.394+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: silver_transformation_pipeline.dbt_run_merchants manual__2025-11-01T14:34:36.691113+00:00 [queued]>
[2025-11-01T14:34:38.395+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 2
[2025-11-01T14:34:38.415+0000] {taskinstance.py:2191} INFO - Executing <Task(BashOperator): dbt_run_merchants> on 2025-11-01 14:34:36.691113+00:00
[2025-11-01T14:34:38.425+0000] {standard_task_runner.py:60} INFO - Started process 569 to run task
[2025-11-01T14:34:38.433+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'silver_transformation_pipeline', 'dbt_run_merchants', 'manual__2025-11-01T14:34:36.691113+00:00', '--job-id', '21', '--raw', '--subdir', 'DAGS_FOLDER/silver_dag.py', '--cfg-path', '/tmp/tmp5dvhrjse']
[2025-11-01T14:34:38.436+0000] {standard_task_runner.py:88} INFO - Job 21: Subtask dbt_run_merchants
[2025-11-01T14:34:38.550+0000] {task_command.py:423} INFO - Running <TaskInstance: silver_transformation_pipeline.dbt_run_merchants manual__2025-11-01T14:34:36.691113+00:00 [running]> on host f19166223143
[2025-11-01T14:34:38.657+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='silver_transformation_pipeline' AIRFLOW_CTX_TASK_ID='dbt_run_merchants' AIRFLOW_CTX_EXECUTION_DATE='2025-11-01T14:34:36.691113+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-11-01T14:34:36.691113+00:00'
[2025-11-01T14:34:38.660+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2025-11-01T14:34:38.662+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'export PATH=$HOME/.local/bin:/usr/local/bin:$PATH; python -m dbt --no-use-colors run --project-dir /opt/***/dbt_project --profiles-dir /opt/***/dbt_project --select silver_merchants --full-refresh']
[2025-11-01T14:34:38.676+0000] {subprocess.py:86} INFO - Output:
[2025-11-01T14:34:38.705+0000] {subprocess.py:93} INFO - /usr/local/bin/python: No module named dbt.__main__; 'dbt' is a package and cannot be directly executed
[2025-11-01T14:34:38.709+0000] {subprocess.py:97} INFO - Command exited with return code 1
[2025-11-01T14:34:38.729+0000] {taskinstance.py:2698} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskinstance.py", line 433, in _execute_task
    result = execute_callable(context=context, **execute_callable_kwargs)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/bash.py", line 212, in execute
    raise AirflowException(
airflow.exceptions.AirflowException: Bash command failed. The command returned a non-zero exit code 1.
[2025-11-01T14:34:38.735+0000] {taskinstance.py:1138} INFO - Marking task as UP_FOR_RETRY. dag_id=silver_transformation_pipeline, task_id=dbt_run_merchants, execution_date=20251101T143436, start_date=20251101T143438, end_date=20251101T143438
[2025-11-01T14:34:38.751+0000] {standard_task_runner.py:107} ERROR - Failed to execute job 21 for task dbt_run_merchants (Bash command failed. The command returned a non-zero exit code 1.; 569)
[2025-11-01T14:34:38.804+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 1
[2025-11-01T14:34:38.857+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
